bufflog.ts

```diff
@@ -44,3 +48,20 @@ export function error(message: string, context?: object) {
 export function critical(message: string, context?: object) {
     pinoLogger.fatal({context: context}, message);
 }
+
+export function middleware() {
```

Comment:
If we define our own `defaults` and allow for an `options` object to be passed here
```suggestion
export function middleware(options = {}) {
```
then we could do something like:

```
    return require('pino-http')({
        ...defaults, 
        ...options
   })
```

This would allow for a lot of flexibility without requiring any extra logic in this package.

Reply:
@esclapes  oh good suggestion! I'll open an other PR for that. I have an idea on how to do that,  but you might have better suggestions